src/main/java/frc/robot/operator_interface/FullOperatorConsoleOI.java

```diff
@@ -57,8 +57,14 @@ public double getTranslateY() {
     return -translateJoystick.getX();
   }
 
+  // temporary replacement as it was in the spot of repel all
```

Comment:
make sure that your button mappings match the software feature sheet and remove any temporary assignments from testing